commons-schemavalidation/src/main/java/eu/europeana/api/commons/JsonValidationConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package eu.europeana.api.commons;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.support.ResourcePatternResolver;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import java.util.List;
+
+@Configuration
+public class JsonValidationConfiguration implements WebMvcConfigurer {
```

Comment:
This class is not required in API-commons, it is to be added in the implementation/calling API where you want to inform the Spring about our JsonSchemaValidatingArgumentResolver.